Lib/random.py

```diff
@@ -189,6 +194,13 @@ def setstate(self, state):
                              "Random.setstate() of version %s" %
                              (version, self.VERSION))
 
+## -------------------- bytes -----------------------------
+
+    # Implementation used by Random and SystemRandom subclasses
+    def _randbytes_getrandbits(self, n):
```

Comment:
`__name__` and `__qualname__` of this method does not look nice.

You can define method with name `randbytes` and then rename it.
```python
def randbytes(self, n): ...
_randbytes_getrandbits = randbytes
del randbytes
```
Or better create an internal `randbytes` function in `__init_subclass__` and set its `__qualname__` (which depends on the class name) manually.

Reply:
Fixed. I also added a test.